tests/query/allocation_policies/test_bytes_scanned_window_allocation_policy.py

```diff
@@ -114,6 +119,9 @@ def test_killswitch(policy: AllocationPolicy) -> None:
         ),
     )
     allowance = policy.get_quota_allowance(tenant_ids)
+    # assert that child methods were not called
+    mock_get_quota_allowance._update_quota_balance.assert_not_called()
```

Comment:
there's a test for the base class, you're changing functionality in the base class. Please put it there.